menoh/mkldnn_with_generic_fallback/context.hpp

```diff
@@ -0,0 +1,72 @@
+#ifndef MENOH_MKLDNN_WITH_FALLBACK_CONTEXT_HPP
+#define MENOH_MKLDNN_WITH_FALLBACK_CONTEXT_HPP
+
+#include <iosfwd>
+
+#include <menoh/any.hpp>
+#include <menoh/array.hpp>
+#include <menoh/node.hpp>
+#include <menoh/optional.hpp>
+
+#include <menoh/mkldnn_with_generic_fallback/logger.hpp>
+#include <menoh/mkldnn_with_generic_fallback/procedure.hpp>
+
+namespace menoh_impl {
+    namespace mkldnn_with_generic_fallback_backend {
+
+        class context {
+        public:
+            optional<std::tuple<procedure, array>>
+            try_to_get_variable(std::string const& name) {
+                return do_try_to_get_variable(name);
+            }
```

Comment:
`try_to_get_variable` returns a procedure for copying, but I couldn't understand from where to where.
Backend-specific memory (e.g. GPU memory) to normal CPU memory?
Also, are those backend-specific memory managed in the array?, or managed separately inside backends?

Reply:
`try_to_get_variable` returns a procedure for copying from Backend-specific memory to normal CPU memory.
Those backend-specific memories are managed separately inside backends.